CHANGELOG.rst

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+Unreleased
```

Comment:
good practice to keep adding to the list vs. our current approach of checking manually for notable updates :D